README.md

```diff
@@ -73,25 +73,24 @@ application, from the *Favorites* or *All Directories* list.
 > In the next steps, you might need the tenant name (or directory name) or the tenant ID (or directory ID). These are 
 presented in the **Properties** of the Azure Active Directory window respectively as *Name* and *Directory ID*
 
-#### Register the app app (Webapp-Openidconnect)
+#### Register the app (Webapp-Openidconnect)
 
 1. In the  **Azure Active Directory** pane, click on **App registrations** and choose **New registration**.
 1. Enter a friendly name for the application, for example 'java-webapp', select "Accounts in any organizational directory 
-and personal Microsoft Accounts (e.g. Skype, Xbox, Outlook.com)" 
-   and select 'Web app / API' as the *Application Type*.
+and personal Microsoft Accounts (e.g. Skype, Xbox, Outlook.com)".
 1. Click **Register** to register the application.
 1. On the left hand menu, click on **Overview** and :
     - copy **Application (client) ID**
     - copy **Directory (tenant) ID**
     - You'll need both of these values later to configure the project, so put them in a safe place
 1. On the left hand menu, click on **Authentication**, and under *Redirect URIs*, select "Web". You will need to enter 
- two different redirect URIs: one for the signIn page, and one for the graph users page. For both, you should use the same
+ two different redirect URIs: one for the signIn page, and one for the graph page. For both, you should use the same
  host and port number, then followed by "/msal4jsample/secure/aad" for the sign in page and 
- "msal4jsample/graph/users" for the users page.
+ "msal4jsample/graph/me" for the user info page.
   By default, the sample uses: 
 
     - `http://localhost:8080/msal4jsample/secure/aad`. 
-    - `http://localhost:8080/msal4jsample/graph/users`
+    - `http://localhost:8080/msal4jsample/graph/me`

```

Comment:
If we change the redirect URIs, we'll need to update them in the Microsoft Doc quickstart as well

Reply:
Ok will update in quickstart also.